api/services/advanced_prompt_template_service.py

```
@@ -27,6 +27,13 @@ def get_prompt(cls, args: dict) -> dict:
             return cls.get_baichuan_prompt(app_mode, model_mode, has_context)
         else:
             return cls.get_common_prompt(app_mode, model_mode, has_context)
+    
+    @classmethod
+    def get_get_account_refresh_token_key(cls, account_id):
+        result = AccountService._get_account_refresh_token_key(account_id)
```

Ruff (0.8.0)

`AccountService` may be undefined, or defined from star imports

(F405)